verify/build-secrets

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -euf
+
+BASE=${1:-./secrets}
+tar -czvf cockpit-verify-secrets.tgz -C "$(dirname $BASE)" "$(basename $BASE)"
```

Comment:
Where is this tarball being used,  I don't see it anywhere else? Is that ever cleaned up again? (This should be done with `shred -u` as it contains a copy of the user's ssh private key).